system/Session/Session.php

```diff
@@ -317,35 +379,35 @@ public function set($data, $value = null)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Get any user data that has been set in the session.
+	 * Get user data that has been set in the session.
 	 *
+	 * If the property exists as "normal", returns it.
+	 * Otherwise, returns an array of any temp or flash data values with the
+	 * property key.
+	 * 
 	 * Replaces the legacy method $session->userdata();
 	 *
-	 * @param null $key
-	 *
-	 * @return array|null
+	 * @param  $key	Identifier of the session property to retrieve
+	 * @return array|null	The property value(s)
 	 */
 	public function get($key = null)
 	{
 		if (isset($key))
 		{
 			return isset($_SESSION[$key]) ? $_SESSION[$key] : null;
-		}
-		elseif (empty($_SESSION))
+		} elseif (empty($_SESSION))
```

Comment:
Wow. I had forgotten PSR2 actually mentioned that. I also had forgotten how many little things I disagreed with in PSR2, though I guess it's mostly the stuff in Section 5 I dislike.  But, in general, I agree with you. 

And, I believe I read about that on PHP The Right Way, IIRC.